superset-frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -16,43 +16,12 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { CSSProperties } from 'react';
-import { styled } from '@superset-ui/core';
-import { CheckboxChecked, CheckboxUnchecked } from 'src/components/Checkbox';
+import { Checkbox as AntCheckbox } from 'antd-v5';
+import type {
+  CheckboxProps as AntCheckboxProps,
+  CheckboxChangeEvent as AntCheckboxChangeEvent,
+} from 'antd-v5/lib/checkbox';
```

Comment:
For future reference if we can import from "antd-v5/es" instead we should do that. [Here](https://github.com/ant-design/ant-design/discussions/42660#discussioncomment-6012700)'s the reference (for treeshaking).

```suggestion
} from 'antd-v5/es/checkbox';
```